website/docs/r/pi_network_interface.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_pi_network_interface"
+description: |-
+  Manages pi_network_interface.
```

Comment:
could you elaborate pi_network_interface?

Reply:
I kept this standard as any other service does.

Line 11 is missing a url to the doc, but that will be added later. The doc team hasn't publish it yet.

Reply:
ok